docs/tutorials/articles/3rd_party_components/src/example.py

```diff
@@ -41,4 +41,11 @@ def folium_map():
 
     
 Gui.register_content_provider(Map, expose_folium)
+
+# import taipy.gui.builder as tgb
+
+# with tgb.Page() as page:
+#     ...
+#     tgb.part(content="{folium_map()}")
+
```

Comment:
To be removed?

Reply:
This is done to have TGB and Markdown here